### DISCUSSION TITLE
One command to clear every data of a player

### DISCUSSION THREAD

**Discussion by MiraiMat** [BG-Software-LLC/SuperiorSkyblock2#1379] [Denied Feature Requests] [answered]
Hey ! 

I would like to know if you can make some command for delete player data on superioskyblock2,

With the uuid or username 

It would be great.

**Comment by OmerBenGera** [comment, marked as answer]
Close the server, then delete the datastore folder inside the folder of the plugin.

**Reply by MiraiMat** [reply]
So, if I would delete only one player ?
And i didn't found datastore

**Reply by Tptuaazn** [reply]
You don't have one if you're using an outdated plugin version.

**Reply by MiraiMat** [reply]
I'm using the 2022.8-b826

**Reply by Tptuaazn** [reply]
Then it will have datastore `folder`.

**Reply by MiraiMat** [reply]
![image](https://user-images.githubusercontent.com/64833499/192857105-278b27b5-92b0-4250-a071-f9030c25bfaa.png)

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
